### DISCUSSION TITLE
I3Blocks strange xkb_layout block alignment

### DISCUSSION THREAD

**Discussion by fly0utWest** [i3/i3#5937] [Q&A]
Hello, recently I stumbled onto strange module behavior: xkb_layout aligned itself a little higher than another ones. Commenting/changing markup and font properties did nothing, any ideas?

```
[wifi]
command=$HOME/.config/i3blocks/wifi/wifi
label=wifi:
INTERFACE=wlo1
interval=60

[volume]
command=$HOME/.config/i3blocks/volume/volume
#LABEL=♪ 
LABEL=VOL 
interval=once
signal=10
#STEP=5%
#MIXER=[determined automatically]
#SCONTROL=[determined automatically]


[battery2]
command=$HOME/.config/i3blocks/battery2/battery2
markup=pango
interval=30

[time]
command=$HOME/.config/i3blocks/time/time
interval=1
#TZ_FILE=~/.tz
STRFTIME_FORMAT=%d-%m-%Y %H:%M
TZONES=$DEFAULT_TZ
TZ_LABELS=Unknown

[xkb_layout]
command=$HOME/.config/i3blocks/xkb_layout/xkb_layout
font=monospace
font_weight=bold
interval=persist
markup=pango
```

![image](https://github.com/i3/i3/assets/93532590/c4f94448-b654-4ab5-91cf-0a027f8c95f2)

**Comment by orestisfl** [comment]
`$HOME/.config/i3blocks/xkb_layout/xkb_layout` is perhaps outputting some special characters instead of `US`. Check that script and if you don't see anything in there, share the script here.
